ARCs/arc-0019.md

```diff
@@ -42,61 +44,107 @@ An Asset conforming this specification **MUST** have:
 
 2. **An "ipfscid" _template_ argument in place of the normal CID.**
 
-Where the format of templates are {*template type* [':' delimited parameters...])
+Where the format of templates are `{<template type>:<parameters...>])`
 
 The ipfscid template definitions is based on properties within the IPFS CID spec: https://github.com/multiformats/cid  
 
-```ipfscid:{version}:{multicodec content-type name}:[field name containing 32-byte digest, ie reserve]:{hash type}```
+```
+ipfscid:<version>:<multicodec content-type name>:<field name (containing 32-byte digest, i.e., "reserve")>:<hash type>
+```
 
-> The intent is to recompose a complete CID based on the content-hash contained within the 32-byte reserve address, but using the correct multicodec content type, ipfs content-id version, and hash type to match how the asset creator will seed the IPFS content.  If a single file is added using the 'ipfs' CLI via 'ipfs add --cid-version=1 metadata.json' then the resulting content will be encoded using the 'raw' multicodec type.  If a directory is added containing one or more files, then it will be encoded using the dag-pb multicodec.  CAR content will also be dag-pb.  Thus based on the method used to post content to IPFS, the ipfscid template should match. 
+> The intent is to recompose a complete CID based on the content-hash contained within the 32-byte reserve address, but using the correct multicodec content type, ipfs content-id version, and hash type to match how the asset creator will seed the IPFS content.  If a single file is added using the 'ipfs' CLI via `ipfs add --cid-version=1 metadata.json` then the resulting content will be encoded using the 'raw' multicodec type.  If a directory is added containing one or more files, then it will be encoded using the dag-pb multicodec.  CAR content will also be dag-pb. Thus based on the method used to post content to IPFS, the ipfscid template should match. 
 
 The parameters to the template ipfscid are:
-1. IPFS _version_, **MUST** be '0' or '1' representing the IPFS CID version.
-2. _Multicodec content-type name_, **MUST** be an IPFS multicodec name of 'raw' or 'dag-pb'.  Other codecs **SHOULD** be supported but are beyond the scope of this proposal.
-3. _Field name_ **MUST** be 'reserve' to represent the reserve address is used for the 32-byte hash but is specified here so future iterations of the specification may allow other fields or syntaxes to reference other mutable field types.
-4. _Hash type_ **MUST** be 'sha2-256' but is explicitly specified so as to allow future versions the option of using other hashes. 
+1. IPFS `<version>`, **MUST** a valid IPFS CID version. Client implementation **MUST** support '0' or '1' and **SHOULD** support future version.
```

Comment:
..must 'be' a valid.. - be was removed accidentally here.
All other changes are excellent, thanks Fabrice!